Jenkinsfile

```diff
@@ -47,7 +47,7 @@ pipeline {
 
     options {
         // Configure an overall timeout for the build of 4 hours.
-        timeout(time: 4, unit: 'HOURS')
```

Comment:
we should drop this commit (if the current duration is less than 4 hours)

Reply:
Build 51 (the only one green at this stage ) took over 7 hours to complete. 
4 hours was appropriate with surefire forkReuse=true